### PR TITLE
Add header block

### LINKED ISSUE #1: Header Block
Noticeably missing is the Header block (`type=header`). Any chance you can add this in? 

I tried adding my own block type locally but I was getting blocked by the validators (either because of the hardcoded type lists or only because you only search the extension assembly for validators)

### PR DESCRIPTION
**Pull request by DOMZE**

### CODE REVIEW

**Review by pinkfloydx33 — CHANGES_REQUESTED** [review]
It appears you've missed updating the validator classes. For example see here:

https://github.com/webmotions/Azure.WebJobs.Extensions/blob/67bbe86a3359861decac27fcc048b6b7a8d644c3/src/WebMotions.Azure.WebJobs.Extensions.Slack/Validators/SlackMessageValidator.cs#L21-L26

This is not the only location with such checks... Just an example

File: src/WebMotions.Azure.WebJobs.Extensions.Slack/Models/Blocks/SlackHeaderBlock.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        public string Type { get; } = "header";
+
+        /// <summary>
+        /// Gets or sets the text for the block
+        /// </summary>
+        /// <remarks>
+        /// Maximum length is 150 characters. 
+        /// </remarks>
+        [Newtonsoft.Json.JsonConverter(typeof(SlackTextConverter))]
+        public SlackText Text { get; set; }
```
Comment (pinkfloydx33) [inline src/WebMotions.Azure.WebJobs.Extensions.Slack/Models/Blocks/SlackHeaderBlock.cs]: Header only supports Plain Text

**Review by pinkfloydx33 — CHANGES_REQUESTED** [review]

File: src/WebMotions.Azure.WebJobs.Extensions.Slack/Models/Blocks/SlackHeaderBlock.cs (hunk 1 above)
Comment (DOMZE) [reply src/WebMotions.Azure.WebJobs.Extensions.Slack/Models/Blocks/SlackHeaderBlock.cs]: Right. Good catch ;-)

**Comment by DOMZE** [comment]
I've checked and the checks for the blocks is only done in the validator you've mentioned. I've updated the PR

**Review by pinkfloydx33 — APPROVED** [review]
LGTM

**Pull request merged**